BookPlayer/Utils/ThemeViewModel.swift

```diff
@@ -13,6 +13,8 @@ import Themeable
 
 class ThemeViewModel: ObservableObject, Themeable {
   @Published var theme: SimpleTheme
+  @Published var defaultArtwork: Image?
+  @Published var defaultUIArtwork: UIImage?
```

Comment:
❓ Any reason to store both? Can you just have `@Published var defaultArtwork: UIImage?` and just use `Image(uiImage: defaultArtwork)` when need be.

`theme.defaultUIArtwork` seems to only be used once so you can also do the opposite and use `ImageRenderer(content: defaultArtwork).uiImage` or try to get rid of it altogether. I feel in the meantime storing only the `UIImage` is probably best.

Reply:
Nice! I wasn't aware of the `ImageRenderer`